kafka-ballerina/tests/secrets/certsandkeys/client.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Shall we update the expiry date of this certificate? Since it is used for tests, at least for 10 years.

Reply:
fixed in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/240/commits/33e42d4cea453cb1de7e96f87402b922c960fd63